deltacat/compute/compactor/utils/round_completion_file.py

```diff
@@ -55,3 +52,24 @@ def write_round_completion_file(
     )
     logger.info(f"round completion file written to: {round_completion_file_s3_url}")
     return round_completion_file_s3_url
+
+
+# write rcf given s3 url
+def write_repartition_completion_file(
+    repartition_completion_file_s3_url: str,
```

Comment:
It is not necessary that this method to be used only for repartition usecase. Any round completion file could be uploaded to s3 using this method.